src/python/tests/core/bot/tasks/fuzz_task_test.py

```diff
@@ -1368,14 +1391,29 @@ def setUp(self):
     # One trial profile for the session.
     self.mock.random.side_effect = [0.3, 0.3]
     self.mock.pick_gestures.return_value = []
+    self.mock.get_component_list.return_value = [{
+        'component': 'component',
+        'link_text': 'rev',
+    }]
     self.mock.current_timestamp.return_value = 0.0
 
-    # Mock out threads with immedietely ending dummies.
-    self.process = mock.MagicMock()
-    self.mock.get_process.return_value = self.process
-    thread = mock.MagicMock()
-    self.process.return_value = thread
-    thread.is_alive.return_value = False
+    crash_queue = queue.Queue()
```

Comment:
Can we just not mock get_queue if we're gonna expect it to return a real Queue?

Reply:
Not sure if you meant that, but I now simplified it a bunch by just switching things to threading.Thread. Another way would be to mock is_trusted_host=True, which has the same effect, but it also activates a few other code paths, which looks like something that should be tested in separation.

Using multiprocessing would be complicated as the fake filesystem (and maybe other testing setup) isn't properly initialized in the other processes.